presto-docs/src/main/sphinx/connector/memory.rst

```diff
@@ -0,0 +1,67 @@
+================
+Memory Connector
+================
+
+The Memory connector stores all data and metadata in RAM on workers
+and both are discarded when Presto restarts.
+
+.. warning::
+
+    This connector is in early experimental stage it is not recommended
+    to use it in a production environment. The intended use of this
+    connector at this point is purely for testing.
+
+.. warning::
+
+    After ``DROP TABLE`` memory is not released immediately. It is released
+    after next write access to memory connector.
+
+.. warning::
+
+    When one worker fails/restarts all data that were stored in it's
+    memory will be lost forever. In other words in case of worker
+    crash/stop/restart there will be partial and silent data loss.
+    If all worker nodes fail/restart when coordinator is keep
+    running tables will end up empty.
```

Comment:
In long term we are planning to add those missing features (like memory limits, memory tracking, data loses) to use it reliably on production.

For now it's either for experimental use on production and for benchmarking. We want to restore micro jmh benchmarks using sql queries, similar to old `AbstractBenchmark`, but without the issue of overhead for tpch generation.